library/foundations/docs/03-layers-and-tokens.mdx

```diff
@@ -44,51 +44,10 @@ When converting Contexts to Layers, the order, precedence and activation of the
 
 ## How to select Dynamic Layers
 
-When consuming Design Tokens, the generated theme implementations should be able to select the correct set of Layers based on the circumstances. This should be enabled by the Design Team throught the use of Layer Parameters.
+When consuming Design Tokens, the generated theme implementations should be able to select the correct set of Layers based on the circumstances. This should be enabled by the Design Team through the use of Layer Parameters.
 
 We proposed two types of parameters, but teams might use different combinations depending on their needs.
 
 _resolution layers_ are controlled by the parameters `screen-min-size` and `screen-max-size`. The layer becomes active if the viewport is between these two values. If the viewport is smaller than the `screen-min-size` or bigger than the `screen-max-size`, the layer is not active.
 
-_section layers_ are controlled by the parameter `section-name`. The layer becomes active within a specific part of the application defined by `section-name` parameter. In web implementations, this is usually defined by either a route or a class name present on either the body or a section tag.
-
-# Design Tokens
```

Comment:
This is in the generated design tokens documentation (and is not correct here) so was removed.